### PR TITLE
Subjects integration

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg [hunk 1]
```diff
@@ -6,9 +6,10 @@
     "iamPublicServer": "aosiam:8090",
     "workingDir": "/var/aos/workdirs/sm",
     "storageDir": "/var/aos/storages",
+    "serviceDir": "/var/aos/workdirs/sm/services",
```
Comment (al1img) [inline recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg:9]: `servicesDir`

File: recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg [hunk 2]
```diff
@@ -6,9 +6,10 @@
...
     "layersDir": "/var/aos/workdirs/sm/layers",
+    "stateDir":  "/var/aos/sm/state",
```
Comment (al1img) [inline recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg:11]: `"stateDir": "/var/aos/states"`
also add:
`"storageDir": "/var/aos/storages"`

File: recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend [hunk 3]
```diff
@@ -34,10 +34,10 @@ do_install_append() {
     install -d ${D}${sysconfdir}/aos
     install -m 0644 ${WORKDIR}/aos_servicemanager.cfg ${D}${sysconfdir}/aos
 
-    sed -i 's/"runner": "runc",/"runner": "${AOS_RUNNER}",/g' ${D}${sysconfdir}/aos/aos_servicemanager.cfg
-
     install -d ${D}${systemd_system_unitdir}
     install -m 0644 ${WORKDIR}/aos-servicemanager.service ${D}${systemd_system_unitdir}
+    install -m 0644 ${S}/src/${GO_IMPORT}/runner/aos-service@.service ${D}${systemd_system_unitdir}
+    sed -i 's/{RUNNER}/${AOS_RUNNER}/g' ${D}${systemd_system_unitdir}/aos/aos-service@.service
```
Comment (al1img) [inline recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend:40]: We removed `{RUNNER}` from service file. We have `runc`.

Comment (Eabramov84) [reply recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend:40]: but in vm we have crun

File: recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg (hunk 2 above)
Comment (Eabramov84) [reply recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg:11]: added

File: recipes-core/images/aos-image-vm.inc [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 BOARD_MODEL ?= "vm-dev;1.0"
 
-BUNDLE_IMAGE_VERSION ?= "2.3.0"
+BUNDLE_IMAGE_VERSION ?= "3.0.0"
```
Comment (al1img) [inline recipes-core/images/aos-image-vm.inc:5]: Probably should be the last commit before release?

File: recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend [hunk 5]
```diff
@@ -34,10 +34,10 @@ do_install_append() {
     install -d ${D}${sysconfdir}/aos
     install -m 0644 ${WORKDIR}/aos_servicemanager.cfg ${D}${sysconfdir}/aos
 
-    sed -i 's/"runner": "runc",/"runner": "${AOS_RUNNER}",/g' ${D}${sysconfdir}/aos/aos_servicemanager.cfg
-
     install -d ${D}${systemd_system_unitdir}
     install -m 0644 ${WORKDIR}/aos-servicemanager.service ${D}${systemd_system_unitdir}
+    install -m 0644 ${S}/src/${GO_IMPORT}/runner/aos-service@.service ${D}${systemd_system_unitdir}
```
Comment (al1img) [inline recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend:39]: Fix commit message.

File: recipes-aos/aos-vis/files/aos_vis.cfg [hunk 6]
```diff
@@ -20,10 +20,10 @@
             }
         },
         {
-            "Plugin": "usersadapter",
+            "Plugin": "subjectsadapter",
             "Params": {
-                "VISPath": "Attribute.Vehicle.UserIdentification.Users",
-                "FilePath": "/var/aos/vis/users"
+                "VISPath": "Attribute.Vehicle.SubjectsIdentification.Subjects",
+                "FilePath": "/var/aos/subjects"
```
Comment (al1img) [inline recipes-aos/aos-vis/files/aos_vis.cfg:26]: move it under vis folder as it is vis related.

**Review by MykolaSolyanko — APPROVED** [review]

**Pull request merged**